microSALT/store/db_manipulator.py

```diff
@@ -152,16 +154,30 @@ def purge_rec(self, name, type):
         self.session.commit()
     self.logger.info("Removed information for {}".format(name))
 
-  def query_rec(self, table, filters):
-    """Fetches records table, by applying a dict with columns as keys."""
-    table = eval(table)
-    args = list()
-    for k,v in filters.items():
-      if v != None:
-        args.append("table.{}=='{}'".format(k, v))
-    filter = ' and '.join(args)
-    entries = self.session.query(table).filter(eval(filter)).all()
-    return entries
+  def query_rec(self, tablename, filters):
```

Comment:
It is a little confusing what types the arguments are expected to be here. Especially `tablename`. It seems it can be some kind of table object, or a string with valid python code? Maybe it would be a good idea to specify this in the function docstring (This is a general comment for all code)